mobile/src/components/SelectableChip.tsx

```diff
@@ -0,0 +1,50 @@
+import { Pressable, Text } from "react-native";
+import Ionicons from "@expo/vector-icons/Ionicons";
+
+interface Props {
+    label: string;
+    selected: boolean;
+    onSelect: (style: string, selected: boolean) => void;
+    className?: string;
```

Comment:
className is reserved on RN and cannot be passed as prop. 

Rename it to style like on the Card component

Reply:
I think @rob-3 figured out how to make the component work with `Classname = ""` but it will not be called a className prop but instead it used a import from nativewind. see: [](https://github.com/poop-team/daydream/blob/main/mobile/src/components/Card.tsx) in the props of the button component

Reply:
Yeah I was following a bit of what you and Victor were doing so I added the className but lmk if I should change

Reply:
yeah calling className in the props there just doesn't work. Sorry I forgot to mention yesterday that the Button component changed

Reply:
Oh ok no worries, I'll change it. SO I can use the Card component as reference?